correios/update_wsdl.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+MODULE_PATH = os.path.join(os.path.dirname(__file__), 'wsdls')
+
+
+WSDL_PATH = os.environ.get('CORREIOS_WSDL_PATH', MODULE_PATH)
```

Comment:
@lamenezes we're using a different lib. I think this part should be agnostic.

Reply:
I don't like to pass arguments to a CLI using envvars instead of `argparse`. Beware that I think it's ok to use envvars as a fallback for comand line arguments (eg. python interpreter uses `-u` for unbuffered output and `PYTHONUNBUFFERED` envvar).

Reply:
With `prettyconf` i'm going to add a new dependency just for this. Are you sure that you want it anyway?

Reply:
@osantana actually the method `update_wsdl` can be imported and called by the library users. So we can give some freedom on how and when the WSDL are updated.
I can add the `argparse` into the script file, and pass this path to the function by argument.

Reply:
IMHO libraries must implement functions/classes/methods that receive all configurations via parameters. Libraries never need to look at envvars either. If I want to use update_wsdl as a module the correct way to do this will be:


```python
from wsdl import update

try:
    update("foo"os.path.join(...))
except UpdateException:
    print("Cannot update foo WSDL")
```

And wsdl.py module must be implemented like:

```python

_default_path = os.path.join(os.path.dirname(__file__), 'wsdls')

def update(path=_default_path):
    # download wsdls in path

def cli():
    args = argparse....
    try:
        path = args.path or os.environ["..."]
    except KeyError:
         # ... error ...

    update(path)

if __name__ == "__main__":
    cli()

Reply:
I'm adding the wsdl_path to `Client.__init__()`. But the client is on `scripts/update-correios-wsdl` so Pip can create a symbolic link on `\user\bin\` and provide a shell script easily.